modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -1501,30 +1501,38 @@ public CodegenProperty fromProperty(String name, Property p) {
         }
 
         property.baseType = getSwaggerType(p);
-
+        // A quick fix to support 1 sub array like [[Object]]
```

Comment:
I think this comment, if necessary, should be above your ~7 line addition and not above the entire block for `ArrayProperty`.